core/src/test/java/org/pentaho/platform/engine/core/system/TestShutdownOrderListener.java

```diff
@@ -0,0 +1,42 @@
+package org.pentaho.platform.engine.core.system;
```

Comment:
I did it this way because I saw several other tests of different types of system listeners set up as separate classes, and I wanted to be consistent with other tests.  I am happy to change it if you think it's overall better to have it in one class though!